index.html

```diff
@@ -235,14 +255,25 @@ <h2>Working Demos</h2>
             including arctic views (not supported by most web map tiling systems).
           </p>
         </li>
-        <li><a href="https://geogratis.gc.ca/mapml/static/cubewerx_cgdi_proxy_geomet_directory.html">MapML versions of GeoMet WMS</a>
+        <li><a href="https://geogratis.gc.ca/mapml/static/us_pop_density.html">Vector example</a>
+          <p>
+            This is an HTML document containing map markup inline, representing
+            vector data according to the MapML &lt;feature&gt; proposal. </a>.
+          </p>
+        </li>
+
+        
+<!--        <li><a href="https://geogratis.gc.ca/mapml/static/cubewerx_cgdi_proxy_geomet_directory.html">MapML versions of GeoMet WMS</a>
           <p>
             Direct access to MapML files,
             generated from live CubeWerx Environment and Climate Change data.
             These files can be dragged and dropped onto the <a href="https://geogratis.gc.ca/mapml/">MapML viewers</a>.
           </p>
         </li>
         <li><a href="https://geogratis.gc.ca/mapml/static/cubewerx_cgdi_proxy_wmts.html">Collection of MapML viewers of CubeWerx maps</a>
+        </li>-->
+        <li><a href="https://www.youtube.com/channel/UCKIv6IGwzxPF2HNhvq-VhKA" target="_blank">Maps for HTML is on YouTube</a>
+          <p>The Maps for HTML Community Group has a <a href="https://www.youtube.com/channel/UCKIv6IGwzxPF2HNhvq-VhKA" target="_blank">channel</a> on YouTube.</p>
```

Comment:
This doesn't really belong in the “Working Demos” section.

Maybe add to the “About the Community Group” section? We should also add the Twitter account there, too.

Also, `target="_blank"` is unnecessary. This is a static page, the browser back button works fine if someone needs to get back to the list!

Reply:
OK